komga-webui/src/locales/en.json

```diff
@@ -4,8 +4,8 @@
       "pageText": "{0}-{1} of {2}"
     },
     "dataIterator": {
-      "loadingText": "Loading items...",
-      "noResultsText": "No matching records found"
```

Comment:
"Could not" is the trending way to present errors, and consistent here.

Reply:
This is the vuetify provided strings. How is that trending and consistent?

Reply:
This is the passive form. If the increasingly popular "Could not" is used for errors, it is in the very least consistent.

Reply:
> If the increasingly popular "Could not" is used for errors

do you have anything to back up those claims?

Reply:
Yes, but I am changing it back.